src/post/post.entity.ts

```diff
@@ -11,6 +11,28 @@ export enum PostType {
     Tombstone = 2,
 }
 
+/**
+ * Visibility of a post
+ */
+export enum PostVisibility {
```

Comment:
I don't think this belongs here, it hasn't really got anything to do with our concept of a Post and is specific to a GhostPost - which is all handled in the webhook. As far as I can see this is only used to create the validation schema in zod? I would probably get rid of this entirely and just use zod to explicitly define the type for `visibility`